src/app/app.component.ts

```diff
@@ -6,5 +6,10 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'my-app';
+  gridSize = 9;
+
+  changeGridSizeNum(num: number) {
```

Comment:
super nit: by convention
```suggestion
  setGridSize(num: number) {
```